DrawMat/Views/MainWindow.axaml.cs

```diff
@@ -11,39 +12,95 @@ namespace DrawMat.Views;
 
 public partial class MainWindow : Window
 {
+    private enum Mode { None, DrawingPolyline, Selection }
+    private Mode currentMode = Mode.Selection;
```

Comment:
I think [the naming convention for private fields is to start with an underscore](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions).